Makefile

```diff
@@ -1,32 +1,58 @@
 .DEFAULT_GOAL := help
 FLASK_APP = hushline/__init__.py
+REDIS_URI ?= memory://
+SECRET_KEY ?= 58b8e28a20490574bd0f0fc3dcd5c34ee8469528f900a5c05f1dea9dadb2ff3f
+ENCRYPTION_KEY = dTq0O0iyxFnJAYjbvILZcQK33FGKfDYBPdLoRZJ6dgg=
+DB_NAME ?= hushlinedb
+DB_USER ?= hushlineuser
+DB_PASS ?= dbpassword
+HUSHLINE_DEBUG_OPTS ?= 0
+SQLALCHEMY_DATABASE_URI ?= sqlite:///$(CURDIR)/hushline.db
+REGISTRATION_CODES_REQUIRED ?= False
+FLASK_ENV ?= development
+
+export FLASK_APP REDIS_URI SECRET_KEY ENCRYPTION_KEY DB_NAME DB_USER DB_PASS HUSHLINE_DEBUG_OPTS SQLALCHEMY_DATABASE_URI REGISTRATION_CODES_REQUIRED FLASK_ENV
 
 .PHONY: help
-help: ## Print the help message
+help:
 	@awk 'BEGIN {FS = ":.*?## "} /^[0-9a-zA-Z_-]+:.*?## / {printf "\033[36m%s\033[0m : %s\n", $$1, $$2}' $(MAKEFILE_LIST) | \
 		sort | \
 		column -s ':' -t
 
+.PHONY: install-deps
```

Comment:
This one is def an ideological nit, but I try to keep the "repeatable scripts" of a `Makefile` to the ones that are tedious to type or vary across projects. What is the exact set of lints and tests the project should run? This project vs. another? But in every project with `poetry`, the command `poetry install` is going to be the same for the dev env. So I guess the rule I use is "things that are too long to type" or "things that aren't obvious."

Reply:
👍 Removed from the makefile. We're also explicit about `poetry install` in the docs, so covered.